distributed/core.py

```diff
@@ -132,12 +132,12 @@ class Server(TCPServer):
     default_port = 0
 
     def __init__(self, handlers, max_buffer_size=MAX_BUFFER_SIZE,
-            connection_limit=512, **kwargs):
+            connection_limit=512, ssl_context_creator= create_ssl_context, **kwargs):
         self.handlers = assoc(handlers, 'identity', self.identity)
         self.id = str(uuid.uuid1())
         self._port = None
         self.rpc = ConnectionPool(limit=connection_limit)
-        super(Server, self).__init__(max_buffer_size=max_buffer_size, **kwargs)
+        super(Server, self).__init__(max_buffer_size=max_buffer_size, ssl_options=ssl_context_creator(), **kwargs)
```

Comment:
I still don't understand why we don't just rely on kwarg pass through?  

```python
    def __init__(..., **kwargs):
        super(Server, self).__init__(..., **kwargs)
```

Why do we need to specifically call an `ssl_context_creator` function at all?  Why can't the caller pass in `ssl_options` from the `Schduler(..., ssl_options=...)` constructor and have them trickle through?

Perhaps you can share an example of how you're doing this now and why this change makes things easier?